### PR TITLE
v19.9 release post

### PR DESCRIPTION
**Pull request by robertbaldyga**
Signed-off-by: Robert Baldyga <robert.baldyga@intel.com>

### CODE REVIEW

File: _posts/2019-10-04-ocf-19.9.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## More fine-grained io classification
+In this release we extended list of available io classification criteria
+by few brand new powerful options:
+- Request size
+- Request address
+- File extension
+- Process name
+- Process ID
+This allows users to more precisely define which data they want to cache,
```
Comment (micrakow) [inline _posts/2019-10-04-ocf-19.9.md]: This is not split into 2 lines in 'rich diff' somehow

File: _posts/2019-10-04-ocf-19.9.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+---
+title: "OCF v19.9 and Open CAS Linux v19.9 Release"
+published: false
+permalink: 2019-10-04.html
+tags: [news]
+---
+I'm pleased to anounce that new releases of both Open CAS Framework and
+Open CAS Linux are now officialy available. Let's look what's new in
```
Comment (Deixx) [inline _posts/2019-10-04-ocf-19.9.md]: officially

Comment (robertbaldyga) [reply _posts/2019-10-04-ocf-19.9.md]: done

File: _posts/2019-10-04-ocf-19.9.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+to handle higher bandwidths. We also reduced number of allocations in IO path
+to provide better latency.
+
+## Promotion Policy
+The major new feature of this release is *eviction noise reduction* a.k.a.
+**Promotion Policy**. The **Promotion Policy** is simply a stateful filter
+which decides wheter given request should be inserted into the cache or not.
+There are two promotion policies available, **ALWAYS** and **NHIT**. **ALWAYS**
+is the delfault and it just marks all requests for insertion. That means there
+is no filtering at all and so is provides the same behavior as it was before
```
[the remark this answers is not in the document]
Comment (robertbaldyga) [reply _posts/2019-10-04-ocf-19.9.md]: done

**Pull request merged**